archive/projects2.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Erica Case Projects</title>
+  <link rel="styles/normalize.css" href="icono.min.css">
+  <link rel="stylesheet" href="styles/main.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <link  rel="stylesheet" href="https://fonts.googleapis.com/css?family=Anton|Indie+Flower|Space+Mono">
+</head>
+
+<body>
+  <header>
+    <a  href="index.html" class="home"> <i class="fa">&#xf1b8;</i></a></li>
+    <h1 class="erica">Erica Case: PROJECTS</h1>
+    <a href="projects.html" class="portfolio"> <i class="fa">&#xf0f2;</i></a></li>
```

Comment:
line 14: nit - extra space after <a
line 14 and 16: nit - no space needed between the a tag and the i tag